src/Coverage/Transformer.hs

```diff
@@ -54,14 +55,29 @@ transform modul =
                 |> List.map (\(AST.UppercaseIdentifier s) -> s)
                 |> intercalate "."
 
+        dropEnd :: Int -> [a] -> [a]
+        dropEnd i xs = reverse . drop i . reverse $ xs
+
+        fileRootDirPath :: FilePath
+        fileRootDirPath =
+            filePath
+                |> splitDirectories
+                |> drop 2 -- .coverage/instrumented/
+                |> dropEnd (length moduleName_)
+                |> joinPath
+
+        moduleNameWithFilePath :: String
+        moduleNameWithFilePath =
+            fileRootDirPath </> moduleName
```

Comment:
We should ideally [keep'em separated :notes:](https://www.youtube.com/watch?v=9XmSyEoX9Gg) instead of smashing them together (this creates eg. an  `src/Foo.Bar` abomination which only gets retroactively fixed later in the `elm-coverage` pipeline) but that was too much thinking for my quick hack